nelmio/api-doc-bundle/3.0/config/packages/nelmio_api_doc.yaml

```diff
@@ -0,0 +1,22 @@
+#nelmio_api_doc:
+#    exclude_sections:     []
+#    default_sections_opened:  true
+#    motd:
+#        template:             'NelmioApiDocBundle::Components/motd.html.twig'
+#    request_listener:
+#        enabled:              true
+#        parameter:            _doc
+#    swagger:
+#        api_base_path:        /api
```

Comment:
when we uncomment, i guess this should have a comment line above to say what this is.

Reply:
This is 2.0 actually, it does no longer exist in 3.0 :)